sprint13 - OOP copy/index.js

```diff
@@ -0,0 +1,50 @@
+class Car {
+    constructor(brand, model, color, mileage) {
+        this.brand = brand;
+        this.midel = model;
+        this.color = color;
+        this.mileage = mileage;
+    }
+
+    getDescriptions() {
+        return 'Car [brand = ${this.brand}, model = ${this.model}, color = ${this.color}, mileage = ${this.mileage}]';
+    }
+}
+
+const main = () => {
+    const firstCar = new Car("Volvo", "XC-60", "Black", 234000);
+    const secondCar = new Car("Mazda", "CX-5", "Green", 111000);
+    const thirtCar = new Car("Dacia", "BIGGSTER", "Blue", 20000);
+
+    console.log(firstCar.getDescriptions());
+    console.log(secondCar.getDescriptions());
+    console.log(thirtCar.getDescriptions());
+};
+main();
+
+class theRecingCar extends Car {
```

Comment:
```suggestion
class RacingCar extends Car {
```

clasa incepe cu litera mare.